appengine/bigquery/requirements.txt

```diff
@@ -1 +1 @@
-google-api-python-client
+google-api-python-client==1.4.2
```

Comment:
By pinning the requirements, we can be certain that our users get the versions that have been tested and vetted by us, and aren't surprised by breaking changes in new versions.

The scripts to check and update are for us, so that we can be made aware of new releases and get them tested and released to users.